tests/ai/src/index.ts

```diff
@@ -1,23 +1,23 @@
-import { App } from '@microsoft/teams.apps';
-import { DevtoolsPlugin } from '@microsoft/teams.dev';
+import { App } from "@microsoft/teams.apps";
+import { DevtoolsPlugin } from "@microsoft/teams.dev";
 // :snippet-start: ai-imports
-import { ChatPrompt } from '@microsoft/teams.ai';
-import { OpenAIChatModel } from '@microsoft/teams.openai';
+import { ChatPrompt } from "@microsoft/teams.ai";
+import { OpenAIChatModel } from "@microsoft/teams.openai";
 // :snippet-end:
-import { MessageActivity } from '@microsoft/teams.api';
-import { ConsoleLogger } from '@microsoft/teams.common';
+import { MessageActivity } from "@microsoft/teams.api";
```

Comment:
These should be single quotes. Could you run `fmt`?

Reply:
Adding linting for tests/samples in a diff PR as per out of band discussion